Index: MysqlDemo/src/Test/Test.go
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- MysqlDemo/src/Test/Test.go	(date 1517303534000)
+++ MysqlDemo/src/Test/Test.go	(revision )
@@ -1,31 +1,32 @@
 package main
 
 import (
-	//"AppStart"
-	"fmt"
-	"strconv"
+	"AppStart"
+	//"fmt"
+	//"strconv"
 )
 
 func main() {
-	//AppStart.AppInit()
-	slice1 := make([]string,3,5)
-
-	slice1[0] = "0"
-	slice1[1] = "1"
-	slice1[2] = "2"
-
-
-	fmt.Println(strconv.Itoa(len(slice1)) + "----------"  + strconv.Itoa(cap(slice1)))
-
-	slice2 := slice1
-
-	slice1 = append(slice1,"3")
-	slice1 = append(slice1,"4")
-
-	fmt.Println(strconv.Itoa(len(slice1)) + "----------"  + strconv.Itoa(cap(slice1)))
-	slice1 = append(slice1,"5")
-	fmt.Println(strconv.Itoa(len(slice1)) + "----------"  + strconv.Itoa(cap(slice1)))
-	fmt.Println(strconv.Itoa(len(slice2)) + "----------"  + strconv.Itoa(cap(slice2)))
+	AppStart.AppInit()
+	//slice1 := make([]string,3,5)
+	//
+	//slice1[0] = "0"
+	//slice1[1] = "1"
+	//slice1[2] = "2"
+	//
+	//
+	//fmt.Println(strconv.Itoa(len(slice1)) + "----------"  + strconv.Itoa(cap(slice1)))
+	//
+	//slice2 := slice1
+	//
+	//slice1 = append(slice1,"3")
+	//slice1 = append(slice1,"4")
+	//
+	//fmt.Println(strconv.Itoa(len(slice1)) + "----------"  + strconv.Itoa(cap(slice1)))
+	//slice1 = append(slice1,"5")
+	//slice1 = append(slice1,"6")
+	//fmt.Println(strconv.Itoa(len(slice1)) + "----------"  + strconv.Itoa(cap(slice1)))
+	//fmt.Println(strconv.Itoa(len(slice2)) + "----------"  + strconv.Itoa(cap(slice2)))
 
 
 }
